inventories/group_vars/all

```diff
@@ -260,6 +260,16 @@ terrain:
   log_file: terrain-docker.log
   max_heap: "{{ max_heap.high }}"
 
+image_janitor:
+  host: "{% if use_overlay %}image-janitor{% else %}{{ groups['image-janitor'][0] }}{% endif %}"
+  port:
+  base: "http://{% if use_overlay %}image-janitor{% else %}{{ groups['image-janitor'][0] }}{% endif %}"
```

Comment:
`host` and `base` for image-janitor don't make sense unless you're adding an image-janitor nginx. in any case I'm sure nothing uses these three things elsewhere, since it'd have to be in this PR. I'd suggest removing them; image-janitor isn't primarily an HTTP service so these aren't relevant for generating any configs or such.